dashboard/src/actions/charts.test.tsx

```diff
@@ -82,10 +80,10 @@ describe("fetchChartVersions", () => {
       { type: getType(actions.charts.requestCharts) },
       { type: getType(actions.charts.receiveChartVersions), payload: response },
     ];
-    await store.dispatch(actions.charts.fetchChartVersions("foo"));
+    await store.dispatch(actions.charts.fetchChartVersions("chart-namespace", "foo"));
     expect(store.getActions()).toEqual(expectedActions);
     expect(axiosGetMock.mock.calls[0][0]).toBe(
-      `api/assetsvc/v1/ns/${namespace}/charts/foo/versions`,
+      `api/assetsvc/v1/ns/chart-namespace/charts/foo/versions`,
```

Comment:
the linter would change this to "api/assetsvc..." (with double quotes) but why did you remove the var?

Reply:
That was a mistake (must have been blindly fixing tests). Switch this (and similar ones) back to use `${namespace}` (and use `chart-namespace` for that var).